libcidr-shopify.rb

```diff
@@ -0,0 +1,19 @@
+class LibcidrShopify < Formula
+  desc 'Libcidr is a library that provides utility functions to handle IP addresses and net blocks.'
+  homepage 'https://github.com/Shopify/libcidr-shopify/'
+  url 'http://bazaar.launchpad.net/~fullermd/libcidr/trunk/tarball/302'
+  sha256 'cb3c51ffec7c6dd83dca1e7f3a92852e4febd230624e84457c269bec82432719'
+
+  patch do
+    url "https://gist.githubusercontent.com/fmejia97/ad72a1f2b5d4b77a2921f32d861850b8/raw/fb499d61e4645c04195a84deb293b2f7a2a4720e/mac_os_libcidr_support.patch"
+    sha256 "b0400ea3bff166c189365c283874e0ff8888a6db89f7a0c541eff88dcd1f2b84"
```

Comment:
nice ... is there a reason in the patch that you introduce an extra variable SHLIB_NAME_PREFIX instead of just assigning SHLIB_LINK one step earlier and using its value instead?

Reply:
@sbfaulkner That's true, there is no need for SHLIB_NAME_PREFIX :) I'll update the patch.

Reply:
actually, as we just discussed IRL you can simplify further and eliminate the `if Darwin` conditions since the only place you'll apply this patch *is* on OS X so just always do it